### DISCUSSION TITLE
compile gpuowl in 2023 , kernel 6.4

### DISCUSSION THREAD

**Discussion by landjalan** [preda/gpuowl#271] [Q&A]
hi guys. downloading the source then running "make" does not work due to the following error :

`src/version.h:7:31: error: static assertion failed: VERSION not set
    7 | static_assert(sizeof(VERSION) > 1, "VERSION not set");
`

entering build dir and executing "meson .." also just leads to an error :

`
../meson.build:12:13: ERROR: C++ shared or static library 'amdocl64' not found

A full log can be found at /...gpuowl/gpuowl-master/build/meson-logs/meson-log.txt
WARNING: Running the setup command as `meson [options]` instead of `meson setup [options]` is ambiguous and deprecated.`

i want to try my nvidia 1650 gtx opencl speed, why is the "meson .." command asking for amd graphics driver lib ??

any advice on how to proceed to get gpuowl running is very much appreciated

thanks

**Comment by tdulcet** [comment]
The Makefile uses the `git describe` command to determine the version, so you would need to download GpuOwl with:
```bash
git clone https://github.com/preda/gpuowl.git
```

**Comment by landjalan** [comment]
indeed, that worked. i just downloaded the zip file, which lead to the before mentioned error

THANK YOU !
